### ISSUE TITLE
[BUG] disk not resized after cloning.

### ISSUE THREAD

**Issue by knobik** [danitso/terraform-provider-proxmox#49]
**Describe the bug**
When you clone a VM template and provide a `disk { size: X }`, the disk is not resized to that value.

**To Reproduce**
```
resource "proxmox_virtual_environment_vm" "ubuntu_vm" {
  count = 3
  name        = "test-worker-${count.index + 1}"
  description = "test worker node :)"

  node_name = "pve" // node name (vm host)

  agent {
    enabled = true
  }

  clone {
    vm_id = 101
  }

  disk {
    datastore_id = "ISOs"
    size = 50
  }
}
```

**Expected behavior**
Resize the cloned disk atleast upwards.

**Additional context**
`terraform plan` on already provisioned (eg. 2nd run) vm's shows this:

```
~ disk {
            datastore_id = "ISOs"
            file_format  = "qcow2"
          ~ size         = 40 -> 50 # forces replacement
        }

```

**Comment by luhahn** [comment]
This is fixed in my pull request: https://github.com/danitso/terraform-provider-proxmox/pull/5

**Comment by danitso-dp** [comment]
@knobik I'm closing this issue as the feature is available in v0.4
